evalml/objectives/utils.py

```diff
@@ -17,43 +24,31 @@ def _all_objectives_dict():
     return objectives_dict
 
 
-def _print_objectives_in_table(names):
-    """Print the list of objective names in a table.
-
-    Returns:
-         None
-    """
-    def iterate_in_batches(sequence, batch_size):
-        return [sequence[pos:pos + batch_size] for pos in range(0, len(sequence), batch_size)]
-    batch_size = 4
-    table = Texttable()
-    table.set_deco(Texttable.BORDER | Texttable.HLINES | Texttable.VLINES)
-    for row in iterate_in_batches(sorted(names), batch_size):
-        if len(row) < batch_size:
-            row += [""] * (batch_size - len(row))
-        table.add_row(row)
-    print(table.draw())
-
-
-def print_all_objective_names():
-    """Get all valid objective names in a table.
+def get_all_objective_names():
+    """Get all valid objective names in a list.
 
     Returns:
         None
     """
     all_objectives_dict = _all_objectives_dict()
-    all_names = list(all_objectives_dict.keys())
-    _print_objectives_in_table(all_names)
+    return list(all_objectives_dict.keys())
+
+
+def get_core_objective_names():
+    """Get a list of all valid core objectives."""
```

Comment:
Should this have similar formatting as the other functions, with the `Arguments` and `Returns` args?